### ISSUE TITLE
Apache & PHP not found : Unable to create fresh latest moodle installation in Debian Server

### ISSUE THREAD

**Issue by AlexTeixeira** [bitnami/bitnami-docker-moodle#196]
**Description**

Unable to create a fresh installation of latest moodle version using docker-compose with persistant data in a Debian 11.2 server

**Steps to reproduce the issue:**

1.  Dowload the docker compose file
[docker-compose.yml.txt](https://github.com/bitnami/bitnami-docker-moodle/files/7987920/docker-compose.yml.txt)

2. Run this docker command
docker-compose up -d && docker-compose logs -f
3. Then you should have this error

<img width="1894" alt="image" src="https://user-images.githubusercontent.com/6555080/152187880-7c4adb31-85dc-46c3-a0b8-2ef29ef3547f.png">

I alreay look at some of this repos issues, no one resolve my problem

**Comment by slayer321** [comment]
After going through our docker-compose.yaml file and the screenshot that you share.
The problem it with the way you are mounting volume for moodle

Currently you are doing like this
```
volumes:
      - '/var/ichizen/moodle:/bitnami/moodle'
      - '/var/ichizen/moodle/data:/bitnami/moodledata'
```

So in the screenshot, I can see that you got an error saying 
`"Found 'moodledata' directory inside /bitnami/moodle. Support for this configuration is deprecated and will be removed soon. Please create a new volume mountpoint at /bitnami/moodledata  and copy all its files there."`

this can be easily fix but changing the volume mount ... something like this
```
volumes:
      - '/var/ichizen/moodle:/bitnami/moodle'
      - '/var/ichizen/moodledata:/bitnami/moodledata'
```

**Comment by AlexTeixeira** [comment]
Hello,

Thanks for you feedback, it's remove the error but now I have a new error when moodle create the table in mariadb

<img width="1214" alt="image" src="https://user-images.githubusercontent.com/6555080/152533090-0cc5f1da-706e-4be4-9755-69b2c4f4aa7a.png">

and stop with code 1

**Comment by juan131** [comment]
Hi @AlexTeixeira 

Please, [note the Bitnami team is not actively monitoring GitHub Issues for this project](https://github.com/bitnami/bitnami-docker-moodle#community-supported-solution). This issue will stay open for 20 days to allow the community to contribute, after 15 days without activity the issue will be marked as stale being closed after 5 days.

**Comment by slayer321** [comment]
Hey @AlexTeixeira  ,
I think the issue is with the mariadb image that you are using ..
As I can see that you are using the latest one which is causing the issue here 
You can refer compatibility chart here (http://www.syndrega.ch/blog/#php-and-dbms-compatibility-of-major-moodle-releases)
we can see that moodle 3.11 support mariadb `MAX=10.5` 
and then If we check the mariadb dockerhub page we can see that current latest version is 10.7 

This can be easily solved by changing the mariadb tag to 10.5 or less 

I test your docker-compose.yaml file with mariadb tag as 10.3 and it is working

**Comment by AlexTeixeira** [comment]
Hi @slayer321,

Thanks for the explainations. Is still not working for me because with 10.3 & 10.5 never start in my Debian Server, I need to try the mariadb config without moodle
